synapse/third_party_rules/access_rules.py

```diff
@@ -241,16 +288,19 @@ def check_event_allowed(self, event, state_events):
 
         return True
 
-    def _on_rules_change(self, event, state_events):
+    def _on_rules_change(
+        self, event: EventBase, state_events: Dict[Tuple[str, str], EventBase],
+    ) -> bool:
         """Implement the checks and behaviour specified on allowing or forbidding a new
         im.vector.room.access_rules event.
 
         Args:
-            event (synapse.events.EventBase): The event to check.
-            state_events (dict[tuple[event type, state key], EventBase]): The state of the
-                room before the event was sent.
+            event: The event to check.
+            state_events: A dict mapping (event type, state key) to state event.
+                State events in the room before the event was sent.
+
         Returns:
-            bool, True if the event can be allowed, False otherwise.
+            Whether the event is allowed.
```

Comment:
not a blocker, but honestly I think it's better to spell out what True and False mean.

Reply:
Fair yeah, technically this ends up leaving things to interpretation.